packages/rules/src/checked.ts

```diff
@@ -0,0 +1,32 @@
+import { isNullOrUndefined } from '../../shared';
+
+const checkedValidator = (value: unknown) => {
+  if (isNullOrUndefined(value) || !getCheckedValue(value)) {
+    return false;
+  }
+
+  return !!String(value).trim().length;
+};
+
+function getCheckedValue(value: unknown) {
+  switch (value) {
+    case true:
+    case 'true':
+    case 'false':
+    case false:
+    case 't':
+    case 'f':
+    case 0:
+    case '0':
+    case 1:
+    case '1':
+    case 'on':
+    case 'off':
+    case 'yes':
+    case 'no':
```

Comment:
I failed to mention this in my previous review, one problem with this list is that it includes the "false" or "unchecked" values which don't represent the rule name anymore and will make the rule confusing to use.

I took another look at your examples and there is no way the user selects something other than the options you provided.

The only other possibility is `undefined` which seems to be what you really want to do here. from what I can tell this is too specific to be accepted in vee-validate. There isn't a general guideline into what kind of rules get accepted but I would say "a rule that will be used commonly" and outside of your example I don't see this being used.

I would accept this PR if we tweak the list to be something like:

- `true`
- `on`
- any truthy value (non-empty strings, numbers != 0, objects, arrays, etc...)

The other stuff will have to be dropped, even the string `'false'` will be treated as truthy because that's what it is in JavaScript.

Let me know what do you think?